pkg/apis/sippyprocessing/v1/types.go

```diff
@@ -22,14 +22,18 @@ type TestReport struct {
 	TopFailingTestsWithBug    []*TestResult  `json:"topFailingTestsWithBug"`
 	TopFailingTestsWithoutBug []*TestResult  `json:"topFailingTestsWithoutBug"`
 	BugsByFailureCount        []bugsv1.Bug   `json:"bugsByFailureCount"`
+
+	// JobFailuresByBugzillaComponent are keyed by bugzilla components
+	JobFailuresByBugzillaComponent map[string]SortedBugzillaComponentResult `json:"jobFailuresByBugzillaComponent"`
 }
 
 // SortedAggregateTestsResult
 type SortedAggregateTestsResult struct {
-	Successes          int          `json:"successes"`
-	Failures           int          `json:"failures"`
-	TestPassPercentage float64      `json:"testPassPercentage"`
-	TestResults        []TestResult `json:"results"`
+	Successes          int     `json:"successes"`
+	Failures           int     `json:"failures"`
+	TestPassPercentage float64 `json:"testPassPercentage"`
+	// TestResults holds the values for individual runs of this test sorted by.....
```

Comment:
don't leave me in suspense......i assume they're storted by pass rate?

it's not really individual runs of the test, right?  isn't it an array of tests, each entry being a testname and the totals/percentages(across all runs of that test) for that test name?

Reply:
> don't leave me in suspense......i assume they're storted by pass rate?
> 
> it's not really individual runs of the test, right? isn't it an array of tests, each entry being a testname and the totals/percentages(across all runs of that test) for that test name?

I have no idea. Your API ;)